crawler/configCrawler.json

```diff
@@ -0,0 +1,13 @@
+{
+	"paths":{
+		"exiftoolPath":"/home/thomas/Documents/master/amos/metadata-hub/crawler/exiftool/exiftoolLinux/exiftool",
+		"inputPath":"/home/thomas/Documents/master",
+		"outputPath":"/home/thomas/Documents/master/amos/metadata-hub/crawler/protocol"
+	},
```

Comment:
this here is computer-specific, not gerneric, maybe a problem?

Reply:
These will have to be changed based on the computer we work on. I guess I could remove the paths for the release.

Reply:
if you think its ok or if you fixed that, then feel free to merge.